contracts/src/main/kotlin/com/contractsAndStates/states/GameBoardState.kt

```diff
@@ -138,17 +161,21 @@ data class TileSideIndex(val value: Int) {
             else value - HexTile.SIDE_COUNT / 2)
 
     fun previous() = TileSideIndex(
-            if (value - 1 < 0) HexTile.SIDE_COUNT - 1
+            if (value < 1) HexTile.SIDE_COUNT - 1
             else value - 1)
 
     fun next() = TileSideIndex(
             if (value + 1 < HexTile.SIDE_COUNT) value + 1
             else 0)
+
+    // TODO Confirm that side 0 is between corner 0 and corner 1.
```

Comment:
@rogersanick That's the updated question.

Reply:
This is correct! Side 0 is between corner 0 and 1